tests/e2e/c/interchain_workflow.go

```
@@ -68,7 +68,7 @@ var _ = e2e.DescribeCChain("[Interchain Workflow]", func() {
 			signedTx, err := types.SignTx(tx, signer, senderKey.ToECDSA())
 			require.NoError(err)
 
-			require.NoError(ethClient.SendTransaction(e2e.DefaultContext(), signedTx))
+			_ = e2e.SendEthTransaction(ethClient, signedTx)
```

Why do we ignore this error?